nixos/doc/manual/development/reviewing-contributions.xml

```diff
@@ -0,0 +1,257 @@
+    <itemizedlist>
+      <listitem><para><literal>8.has: package (update)</literal> and any topic label that fit the updated package.</para></listitem>
+    </itemizedlist>
+  </listitem>
+  <listitem><para>Checking that the package versioning is fitting the guidelines.</para></listitem>
+  <listitem><para>Checking that the commit text is fitting the guidelines.</para></listitem>
+  <listitem><para>Checking that the package maintainers are notified.</para>
+    <itemizedlist>
+      <listitem><para>mention-bot usually notify github users based on the submitted changes, but it can happen that it miss some of the package maintainers.</para></listitem>
+    </itemizedlist>
+  </listitem>
+  <listitem><para>Checking that the code contains no typos.</para></listitem>
+  <listitem><para>Building the package locally.</para>
+    <itemizedlist>
+      <listitem><para>Pull-requests are often targetted to the master or staging branch so building the pull-request locally as it is submitted can trigger a large amount of source builds.</para>
+        <para>It is possible to rebase the changes on nixos-unstable or nixpkgs-unstable for easier review.</para>
+<screen>
+TODO: add commands for a local rebase and build
+</screen>
+      </listitem>
+    </itemizedlist>
+  </listitem>
+  <listitem><para>Running every binary.</para></listitem>
+  <listitem><para>Checking that the packages using the updated package are building fine.</para>
+<screen>
+TODO: add commands
+</screen>
+  </listitem>
+</itemizedlist>
+
+<example><title>Sample template for a package update review</title>
+<screen>
+##### Reviewed points
+
+- [ ] package name fits guidelines
+- [ ] package version fits guidelines
+- [ ] package build on ARCHITECTURE
+- [ ] executables tested on ARCHITECTURE
+- [ ] all depending packages build
+
+##### Possible improvements
+
+##### Comments
+
+</screen></example>
+</section>
+
+<section><title>New packages</title>
+
+<para>New packages are a common type of pull-requests. These pull requests consists in adding a new nix-expression for a package.</para>
+
+<para>Reviewing process:</para>
+
+<itemizedlist>
+  <listitem><para>Add labels to the pull-request.</para>
+    <itemizedlist>
+      <listitem><para><literal>8.has: package (new)</literal> and any topic label that fit the new package.</para></listitem>
+    </itemizedlist>
+  </listitem>
+  <listitem><para>Checking that the package versioning is fitting the guidelines.</para></listitem>
+  <listitem><para>Checking that the commit name is fitting the guidelines.</para></listitem>
+  <listitem><para>Checking that the meta field contains correct information.</para>
+    <itemizedlist>
+      <listitem><para>License must be checked to be fitting upstream license.</para></listitem>
+      <listitem><para>Platforms should be set or the package will not get binary substitutes.</para></listitem>
+      <listitem><para>If there is no maintainer, propose the submitter to become the maintainer.</para></listitem>
```

Comment:
> A person MUST become the maintainer of the new package he proposes.

It feels a little too strict and could refrain people from contributing.

I remember a case where a person submitted a package but didn't had the confidence to become the maintainer so he asked me to become the maintainer.

What about something like "The package **must** have a maintainer, this can be the submitter or another community member that accepts to take maintainership of the package."